docs/guides/stay-up-to-date/terraform/how-to-update-to-aws-provider-v4/deployment-walkthrough.md

```diff
@@ -0,0 +1,148 @@
+# Deployment walkthrough
+
+## Updating references to Gruntwork Infrastructure as Code Library
+
+In order to take advantage of the Terraform AWS provider version 4, you need to update your references to the Gruntwork
+Infrastructure as Code Library to use a compatible version. We (Gruntwork) have gone through all our modules in the
+library to test and update the code to be compatible with AWS provider version 4. As a customer, you need to update to
+the proper versions of the Gruntwork library to pick up the fixes/changes that were made to be compatible. Be sure to
+read the release notes to know what changes need to be made to update to that version.
+
+Refer to our guide on ["Updating to new versions"](/guides/working-with-code/versioning#updating-to-new-versions)
+for instructions on how to update the versions in your code.
+
+The following table lists the versions of relevant Gruntwork AWS IaC library modules and Service Catalogs which are
+compatible with AWS provider version 4. Since the AWS provider version 3 upgrade, we have sunsetted some of our module 
+repos. They have not been updated and are not listed below. Please visit the repo READMEs to see our recommendations
+for alternatives to those modules.
+
+:::caution
+
+Gruntwork follows [semantic versioning](/guides/working-with-code/versioning#semantic-versioning).
+For pre-1.0 modules, version updates to the _minor_ version are considered backward incompatible releases. Make sure to
+read the release notes for the relevant modules anytime you are updating minor versions! For example, if you are going
+from `v0.5.x` to `v0.9.x`, read the notes for `v0.6.0`, `v0.7.0`, `v0.8.0`, and `v0.9.0` to get the migration steps for
+all backward incompatible updates in your upgrade path.
+
+:::
+
+## Version Compatibility table
+
+<table>
+<colgroup>
+<col />
+<col />
+</colgroup>
+<tbody>
+<tr className="odd">
+<td><p><strong>Gruntwork Repo</strong></p></td>
+<td><p><strong>Minimum version with AWS Provider v4 support</strong></p></td>
+</tr>
+<tr className="even">
+<td><p>terraform-aws-utilities</p></td>
+<td><p><strong><a href="https://github.com/gruntwork-io/terraform-aws-utilities/releases/tag/v0.9.1">v0.9.0</a></strong></p></td>
```

Comment:
Links to v0.9.1 but text says v0.9.0?

Reply:
Agh, good catch! I'll double-check the rest.

Reply:
Done, no others were incorrect.